packages/i18next/src/lib/types.ts

```diff
@@ -1,8 +1,35 @@
 import type { Awaitable, NonNullObject } from '@sapphire/utilities';
+import type { WatchOptions } from 'chokidar';
 import type { Guild, Message, MessageOptions, StageChannel, StoreChannel, User, VoiceChannel } from 'discord.js';
 import type { InitOptions, StringMap, TFunctionKeys, TOptions } from 'i18next';
 import type { i18nextFsBackend } from 'i18next-fs-backend';
 
+/**
+ * This option is used to set the HMR options.
+ * @since 2.1.6
+ */
+export interface HMROptions {
+	/**
+	 * HMR status for the i18next plugin.
+	 */
+	enabled?: boolean;
```

Comment:
```suggestion
	enabled: boolean;
```

I'd say make this required. The moment people specify `hmr` in client options I'd expect them to at least say whether it is to be enabled or not. Otherwise this would be valid which is really weird:

```ts
{
	i18n: {
		hmr: {}
	}
}
```